tests/integration/classification_test.py

```diff
@@ -4,41 +4,50 @@
 
 from pyatlan.cache.classification_cache import ClassificationCache
 from pyatlan.client.atlan import AtlanClient
+from pyatlan.model.enums import AtlanClassificationColor, AtlanTypeCategory
 from pyatlan.model.typedef import ClassificationDef
 
-CLS_NAME = "psdk-ClassificationTest"
+CLS_NAME = "psdk-classification"
 
-# NOTE: Tests are intended to run in the order specified in this file,
-# to complete creation prior to testing prior to deletion.
 
+class ClassificationTest:
```

Comment:
If you add a class for called ClassificationTest then I would expect the tests to be part of the class not external to it.